_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -424,6 +424,28 @@ https://aws.amazon.com/kms/[KMS]. It can be a good idea to enable CloudTrail in
 multi-region feature enabled, as the API call data is useful useful for troubleshooting, investigating security
 incidents, and maintaining audit logs for compliance.
 
+=== AWS Config
+
+_https://aws.amazon.com/config/[AWS Config]_ is a service that enables you to assess, audit, and evaluate the configurations of
+your https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html[AWS resources]. You can use AWS
+Config to ensure that AWS resources are configured in a manner that is in compliance with your company policies or regulatory
+requirements. This enables you to simplify compliance auditing, security analysis, change management, and operational
+troubleshooting.
+
+AWS Config Rules::
```

Comment:
NIT: since there's only one item here, using a definition list doesn't make sense. Perhaps just transition into this from the previous paragraph via, "The way you specify your company's policies and requirements is by defining _AWS Config Rules_, which are expressions..."

Reply:
Ok fixed.